python/activity_stream/activity_stream.py

```diff
@@ -94,8 +94,12 @@ def __init__(self, parent):
         self._entity_type = None
         self._entity_id = None
 
-        # When a ReplyDialog is active it will be stored here.
-        self.reply_dialog = None
+        self.reply_dialog = ReplyDialog(
```

Comment:
Can we please add a comment here explaining to future code readers that this reply dialog is popped up and positioned whenever a reply ui is needed (by _on_reply_clicked).

Out of interest, what is the benefit of having a single dialog instance that is being reused over making it every time? Now the code feels less contained. 

Heads up that I tried initially building this as inline UI but had issues with pyside crashing, this is why I ended up with this wonky implementation where we pop up a UI and position it on top of the existing UI. Would be great if at some point we would be able to get rid of this dialog popping up altogether!